qa/acceptance/spec/shared_examples/cli/logstash/version.rb

```diff
@@ -14,9 +14,15 @@
 
     context "on #{logstash.hostname}" do
       it "returns the right logstash version" do
-        result = logstash.run_command_in_path("bin/logstash --path.settings=/etc/logstash --version")
+        result = logstash.run_command_in_path("bin/logstash --version")
         expect(result).to run_successfully_and_output(/#{LOGSTASH_VERSION}/)
       end
+      context "when also using the --path.settings argument" do
```

Comment:
+1 for this, you should definitely add this other test too, should not be hard, right?